amundsen_common/models/table.py

```diff
@@ -33,6 +33,19 @@ class Meta:
         register_as_scheme = True
 
 
+@attr.s(auto_attribs=True, kw_only=True)
+class Badge:
+    badge_name: str = attr.ib()
+    badge_type: str = attr.ib()
+    sentiment: str = attr.ib()
```

Comment:
let's still call it category which could be more generic for other use cases.

Reply:
do you plan to `column`, `table status` in category and negative / neutral in badge_type?

Reply:
column, table status in category and negative / neutral in sentiment

Reply:
I thought we are going with badge_type and category?

Reply:
I thought we had said that sentiment was a good descriptor for the color of the badge (positive, negative, neutral), while category would be specific to the type of badge, that's why I renamed category to be badge_type.